iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeSubmitOtpView.swift

```diff
@@ -0,0 +1,162 @@
+                
+                RoundedRectangle(cornerRadius: 10)
+                    .fill(.gray.opacity(0.25))
+                    .frame(width: 300, height: 450)
+                    .padding(.top, 0)
+                
+                VStack {
+                    Image(.msdkPhone)
+                        .resizable()
+                        .blur(radius: 0.0)
+                        .frame(width: 150, height: 150)
+                        .padding(.bottom, 50)
+                        .shadow(color: .black, radius: 3)
+                    
+                    if isAuthenticating {
+                        ProgressView()
+                    }
+                    
+                    if !errorMessage.isEmpty {
+                        Text(errorMessage)
+                            .frame(width: 250, height: 50)
+                            .foregroundStyle(.red)
+                    } else {
+                        Spacer().frame(width: 250, height: 65)
+                    }
+                    
+                    TextField("One-Time-Passcode", text: $otp)
+                        .autocapitalization(.none)
+                        .buttonStyle(.borderless)
+                        .disableAutocorrection(true)
+                        .foregroundColor(.blue)
+                        .frame(maxWidth: 250)
+                        .multilineTextAlignment(.center)
+                        .padding(.top, 25)
+                        .zIndex(2.0)
+                    
+                    Button {
+                        onSubmitOtpTapped()
+                    } label: {
+                        HStack {
+                            Image(systemName: "lock")
+                            Text("Log In Using OTP")
+                        }.frame(minWidth: 150)
+                    }
+                    .buttonStyle(.bordered)
+                    .tint(.blue)
+                    .zIndex(2.0)
+                }
+            }
+            
+            Spacer()
+        }
+        .background(
+            Gradient(colors: [.blue, .cyan, .green]).opacity(0.6)
+        )
+        .blur(radius: isAuthenticating ? 2.0 : 0.0)
+    }
+    
+    ///
+    /// Submits the OTP verification request when the request button is tapped.  This submits a request to
+    /// the `services/oauth2/authorize`endpoint and in turn the `services/oauth2/token`
+    /// endpoint.
+    ///
+    /// On receiving a successful response from the endpoints, the user will be authenticated by Salesforce
+    /// Mobile SDK and returned to the app's views.
+    ///
+    private func onSubmitOtpTapped() {
+        // Reset the error message if needed.
+        errorMessage = ""
+        
+        // Show the progress indicator.
+        isAuthenticating = true
+        
+        // Submit the request and act on the response.
+        Task {
+            // Submit the request.
+            let _ = await SalesforceManager.shared.nativeLoginManager()
```

Comment:
Also, really basic SwiftUI here.  the user is authenticated and returned to the app's view by MSDK at this point.

Reply:
if this is the recommended way to call it, could `@discardableResult` be added to the MSDK method?